playkit/src/main/java/com/kaltura/playkit/PKDrmParams.java

```diff
@@ -58,7 +59,9 @@ protected PKDrmParams(Parcel in) {
     }
 
     public boolean isSchemeSupported() {
-        return scheme != null && scheme.isSupported();
+        boolean isSchemeSupported = (scheme != null && scheme.isSupported());
```

Comment:
Please revert changes in this file -- all they add is a log, which just adds noise.